etc/databases/world/updates/updates.sql

```diff
@@ -30510,6 +30510,47 @@ begin not atomic
         insert into applied_updates values ('080320231');
     end if;
 
+    -- 10/03/2023 1
+    if (select count(*) from applied_updates where id='100320231') = 0 then
+        -- Despawn herbalism nodes not in 0.5.3 alpha
+        -- Dreamfoil
+        update spawns_gameobjects set ignored = 1 where spawn_entry = 176584;
+        -- Black Lotus
+        update spawns_gameobjects set ignored = 1 where spawn_entry = 176589;
+        -- Golden Sansam
+        update spawns_gameobjects set ignored = 1 where spawn_entry = 176583;
+        -- Gromsblood
+        update spawns_gameobjects set ignored = 1 where spawn_entry = 142145;
+        -- Plaguebloom
+        update spawns_gameobjects set ignored = 1 where spawn_entry = 176587;
+        -- Sungrass
+        update spawns_gameobjects set ignored = 1 where spawn_entry = 142142;
+        -- Mountain Silversage
+        update spawns_gameobjects set ignored = 1 where spawn_entry = 176586;
+
+        -- Despawn mining nodes not in 0.5.3 alpha
+        -- Thorium (Rich, Small, Ooze Covered)
+        update spawns_gameobjects set ignored = 1 where spawn_entry in(177388, 123848, 175404, 324, 150082, 176643);
+        -- Truesilver (Deposit, Ooze Covered)
+        update spawns_gameobjects set ignored = 1 where spawn_entry in(123309, 2047, 150081);
+        -- Mithril (Deposit, Ooze Covered)
+        update spawns_gameobjects set ignored = 1 where spawn_entry in(2040, 150079, 176645, 123310);
+
+        -- Despawn chests not available in 0.5.3
```

Comment:
I noticed these chests all contain loot present in 0.5.3. Unless a specific spawn is spawned in a non accessible area etc, I'd not remove them.

Reply:
Since there are no Guardians of Blizzard preventing you from entering unfinished areas you can access *all* of these chests. However, since the areas are clearly unfinished - and often completely devoid of anything - the chests aren't supposed to be there yet.